build/images/wireguard-go/Dockerfile

```diff
@@ -1,33 +0,0 @@
-# Copyright 2022 Antrea Authors
```

Comment:
Why remove this file？

Reply:
oops, I accidentally merge this commit into previous commit. Updated